app/config.go

```diff
@@ -319,9 +319,9 @@ func (a *App) regenerateClientConfig() {
 	}
 
 	clientConfigJSON, _ := json.Marshal(clientConfig)
-	a.Srv().clientConfig = clientConfig
-	a.Srv().limitedClientConfig = limitedClientConfig
-	a.Srv().clientConfigHash = fmt.Sprintf("%x", md5.Sum(clientConfigJSON))
+	a.Srv().clientConfig.Store(clientConfig)
+	a.Srv().limitedClientConfig.Store(limitedClientConfig)
+	a.Srv().clientConfigHash.Store(fmt.Sprintf("%x", md5.Sum(clientConfigJSON)))
```

Comment:
Just want to confirm that we are making sure that this is always going to execute before any call that loads the value (e.g. `ClientConfig()`).

Reply:
Yes, this gets called during server startup from `RunOldAppInitialization` using our wonderful FakeApp adapter.